### PR TITLE
Add a list of Osquery fleet managers

### PR DESCRIPTION
**Pull request by hughneale**
Following on from a quick discussion on Slack. I've added a table to show case the available fleet managers for Osquery.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,6 +111,15 @@ guide](https://osquery.readthedocs.io/en/latest/development/building/). Also
 check out our [contributing guide](CONTRIBUTING.md) and join the
 community on [Slack](https://join.slack.com/t/osquery/shared_invite/zt-h29zm0gk-s2DBtGUTW4CFel0f0IjTEw).
 
+## Osquery fleet managers
+
+| Project                                             | License     |
+| --------------------------------------------------- | ----------- |
+| [Fleet](https://github.com/fleetdm/fleet)           | Commercial  |
```
Comment (zwass) [inline README.md:118]:
```suggestion
| [Fleet](https://github.com/fleetdm/fleet)           | Open Core  |
```

**Review by zwass — DISMISSED** [review]

**Review by Smjert — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -111,6 +111,16 @@ guide](https://osquery.readthedocs.io/en/latest/development/building/). Also
 check out our [contributing guide](CONTRIBUTING.md) and join the
 community on [Slack](https://join.slack.com/t/osquery/shared_invite/zt-h29zm0gk-s2DBtGUTW4CFel0f0IjTEw).
 
+## Osquery fleet managers
+
+| Project                                                 | License     |
+| ------------------------------------------------------- | ----------- |
+| [Fleet](https://github.com/fleetdm/fleet)               | Open Core   |
+| [Kolide](https://www.kolide.com)                        | Freemium    |
```
Comment (directionless) [inline README.md:119]:
```suggestion
| [Kolide](https://www.kolide.com)                        | Commercial    |
```

**Review by directionless — COMMENTED** [review]
I'm a little ambivalent about this. But will to go with the flow. 

I think it's hard to enumerate everyone. And that makes me wonder if this is the wrong place for it.

**Comment by Smjert** [comment]
PS: I don't have any investment here, I just approved because there wasn't anything against it and the change request was resolved.

I think the TFC should decide here.

**Review by directionless — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -111,6 +111,16 @@ guide](https://osquery.readthedocs.io/en/latest/development/building/). Also
 check out our [contributing guide](CONTRIBUTING.md) and join the
 community on [Slack](https://join.slack.com/t/osquery/shared_invite/zt-h29zm0gk-s2DBtGUTW4CFel0f0IjTEw).
 
+## Osquery fleet managers
+
```
Comment (directionless) [inline README.md:115]:
```suggestion
## Osquery fleet managers

There are many osquery fleet managers out there. The osquery project does not endorse, recommend, or test these. They are provided as a starting point
```

File: README.md [hunk 4]
```diff
@@ -111,6 +111,17 @@ guide](https://osquery.readthedocs.io/en/latest/development/building/). Also
...
+There are many osquery fleet managers out there. The osquery project does not endorse, recommend, or test these. They are provided as a starting point
+| Project                                                 | License     |
```
Comment (directionless) [inline README.md:117]:
```suggestion
There are many osquery fleet managers out there. The osquery project does not endorse, recommend, or test these. They are provided as a starting point

| Project                                                 | License     |
```

**Comment by alessandrogario** [comment]
> I'm a little ambivalent about this. But will to go with the flow.
> 
> I think it's hard to enumerate everyone. And that makes me wonder if this is the wrong place for it.

Not against it, just offering an opinion: Kolide and Fleet are both upstream contributors and ensure that osquery is well tested against those environments. From the point of view of a user, it may be useful to know this detail when choosing a fleet manager. Without this, it's not immediately clear whether we are endorsing software we tried/develop or just listing 3rd party projects we are not involved in (which may be useful to disclose)

**Comment by directionless** [comment]
> > I'm a little ambivalent about this. But will to go with the flow.
> > I think it's hard to enumerate everyone. And that makes me wonder if this is the wrong place for it.
> 
> Not against it, just offering an opinion: Kolide and Fleet are both upstream contributors that ensure osquery is well tested against those environments. From the point of view of a user, it may be useful to know this detail when choosing a fleet manager. Without this, it's not immediately clear whether we are endorsing software we tried/develop or just listing 3rd party projects we are not involved in (which may be useful to disclose)

I added some text. How's it look to you?

**Comment by directionless** [comment]
> If you think it's worth mentioning that both Fleet and Kolide are active upstream contributors (contributing not just patches but also a lot of testing at each release) they could maybe be grouped in their own category. Otherwise, this looks good!

That seemed like more effort than it's worth right now.

**Review by directionless — APPROVED** [review]

**Pull request merged**